pytorch_lightning/loops/epoch/prediction_epoch_loop.py

```diff
@@ -4,7 +4,7 @@
 import torch
 from deprecate import void
 
-from pytorch_lightning.loops.base import Loop
+from pytorch_lightning.loops.loop import Loop
```

Comment:
Let's add a 1 cycle depreceation. Some projects are relying on the `from pytorch_lightning.loops.base import Loop`